tests/iulianExample.spec.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('coolors.co has the correct title', async ({ page }) => {
+  // Click the get started link.
+  await page.goto('https://coolors.co/');
+
+  // Expect a title "to contain" a substring.
```

Comment:
Congrats @Iulian99  for your test, it looks good 👍 

Would you agree the the above comment does not add a lot of value if we look at the code below (line 9).
The code you wrote here is intuitive and can be read in natural language to figure out what it does.

If you agree then please remove comments that you consider redundant. 

Feel free to also check out https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/ (especially the cartoons at the end :P )